src/PowerShellEditorServices.Hosting/EditorServicesLoader.cs

```diff
@@ -406,5 +406,28 @@ private void ValidateConfiguration()
                 throw new ArgumentNullException(nameof(_hostConfig.PSHost));
             }
         }
+
+#if !CoreCLR
+        private static void LoadAllAssembliesInDirectory(HostLogger logger, string directoryPath)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'LoadAllAssembliesInDirectory'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5155661)